package.json

```diff
@@ -99,7 +100,7 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "leaflet": "^1.6.0"
```

Comment:
I didn't explicitly remove Leaflet from dependencies - I believe that this happened automatically when I ran `npm install` while using a fairly new version of NPM - probably because Leaflet is also listed under `optionalDependencies`?

Reply:
Yeah, the new npm versions fix that. If a dependency is listed under optional or peer dependencies, then it shouldn't be listed under dependencies. The change is correct. 👍